<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QGroupBox" name="groupBox_5" >
      <property name="title" >
       <string>Connection</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="connectionConnect" >
           <property name="text" >
            <string>Connect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="connectionDisconnect" >
           <property name="text" >
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="connectAddress" >
           <property name="text" >
            <string>localhost</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="protocolLabel" >
         <property name="text" >
          <string>protocol unknown</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox" >
      <property name="title" >
       <string>Pose</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QDoubleSpinBox" name="poseX" >
           <property name="value" >
            <double>13.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="poseY" >
           <property name="value" >
            <double>13.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="poseAngle" >
           <property name="minimum" >
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="poseGet" >
           <property name="text" >
            <string>get</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="poseSet" >
           <property name="text" >
            <string>set</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6" >
      <property name="title" >
       <string>Motion</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QDoubleSpinBox" name="moveX" >
           <property name="minimum" >
            <double>-100.000000000000000</double>
           </property>
           <property name="value" >
            <double>25.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="moveY" >
           <property name="minimum" >
            <double>-100.000000000000000</double>
           </property>
           <property name="value" >
            <double>15.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="moveAngle" >
           <property name="minimum" >
            <double>-720.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>720.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveSpline" >
           <property name="text" >
            <string>moveSpline</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QDoubleSpinBox" name="moveDistance" >
           <property name="minimum" >
            <double>-500.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>500.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveTranslate" >
           <property name="text" >
            <string>translate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QDoubleSpinBox" name="moveAngleDiff" >
           <property name="minimum" >
            <double>-720.000000000000000</double>
           </property>
           <property name="maximum" >
            <double>720.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="moveRotate" >
           <property name="text" >
            <string>rotate</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="moveStop" >
         <property name="text" >
          <string>stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_7" >
      <property name="title" >
       <string>CollisionAvoidance</string>
      </property>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="caEnable" >
         <property name="text" >
          <string>enable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="caDisable" >
         <property name="text" >
          <string>disable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="caStatus" >
         <property name="text" >
          <string>enabled</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="title" >
       <string>RobotStatus</string>
      </property>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QPushButton" name="robotStatusGet" >
         <property name="text" >
          <string>getStatus</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="robotStatusLabel" >
         <property name="text" >
          <string>unknown</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="title" >
       <string>Scanner</string>
      </property>
      <layout class="QHBoxLayout" >
       <item>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QPushButton" name="scannerGet" >
           <property name="text" >
            <string>getScan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="scannerNumber" />
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="scannerView" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="columnCount" >
          <number>2</number>
         </property>
         <column>
          <property name="text" >
           <string>scanNumber</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>scanDistance</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="title" >
       <string>Voltage &amp;&amp; Temps</string>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="voltageGet" >
           <property name="text" >
            <string>Voltage</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="voltageLabel" >
           <property name="text" >
            <string>unknown</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QPushButton" name="tempsGet" >
           <property name="text" >
            <string>Temps</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="tempsLabel" >
           <property name="text" >
            <string>unknown</string>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources/>
 <connections/>
</ui>
